src/Schemes/AffineSchemes/Objects/Attributes.jl

```diff
@@ -202,72 +396,32 @@ julia> (x1,x2,x3) = gens(R)
 julia> Y = subscheme(X,ideal(R,[x1*x2]))
 Spec of Quotient of Multivariate Polynomial Ring in x1, x2, x3 over Rational Field by ideal(x1*x2)
 
-julia> defining_ideal(Y)
+julia> ambient_closure_ideal(Y)
```

Comment:
For better understanding of the setting, maybe add another line for the base_ring of this ideal.

Reply:
I don't understand what you mean here.

Reply:
We are in the documentation here. the returned ideal lives in R. Maybe stress this by adding a line checking that the returned ideal is an ideal in R -- for clarifying to the user, where the object lives.